.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: d
 d:
   - dmd-nightly
-  - dmd-beta
```

Comment:
What's the reason for this? Just that it takes too long? Doesn't it make more sense to test with the beta rather than the nightly build?

Reply:
I'm not sure, I just followed what @BBasile did without much thinking (I still haven't clear idea which one is right, I doubt it matters).

Reply:
@BBasile ?

Reply:
testing with dmd beta is only useful a few weeks per years. Example is now. 2.081-beta.1 (and .2 now) is out but after 2.081.0 released Travis will continue testing the old beta while the matching final release is out which is absurd.

Reply:
OK, that sounds reasonable. Can you put that in the commit message @nemanja-boric-sociomantic ?

Reply:
Sure.